scout/server/blueprints/variant/controllers.py

```diff
@@ -144,6 +145,19 @@ def variant(
     if add_case:
         variant_case(store, case_obj, variant_obj)
 
+    # Fetch ids for grouped cases and prepare alignment dispay
+    case_groups = {}
+    case_group_label = {}
```

Comment:
Do we really need this variable? I don't see it used later in the templates 🤔

Reply:
No, the label is used on the case pages to organise the groups, but it is not used downstream really - yet. It should be safe to remove it!